bin/bundix

```diff
@@ -1,92 +1,97 @@
-#!/usr/bin/env ruby
+#! /usr/bin/env ruby
```

Comment:
I just think it's more readable this way, don't care much either way, this is a change left over from an experiment using the `nix-shell` shebang for ruby, which unfortunately is still blocked by https://github.com/NixOS/nix/issues/877 with a pending PR.